### ISSUE TITLE
Creating docs.

### ISSUE THREAD

**Issue by omatamix** [demency/laravel-friendships#13]
I think it would be better to make a docs on how to install and use this package rather than having a long readme to scroll through.

**Comment by demency** [comment]
I agree with you, we need search a easy-to-use solution for this. You know some path to solve this?
